src/main/java/org/opentripplanner/middleware/tripMonitor/JourneyState.java

```diff
@@ -75,17 +67,16 @@
     public JourneyState() {}
 
     /**
-     * Update journey state based on results from {@link CheckMonitoredTrip}.
-     * TODO: This may need some tweaking depending on whether a check was successfully completed or not.
-     *   E.g., should a previous journey state be overwritten by a failed check?
+     * Clone this object.
+     * NOTE: This is used primarily during testing and only clones certain needed items so not all entities are
+     * deep-cloned. Implement this further if additional items should be deep-cloned.
```

Comment:
This is no longer just used in testing, right? It looks like this gets called in the CheckMonitoredTrip constructor now.

Reply:
Removed note about testing in 74d09031e8b0f80d0116218d4cb6cfb4d509d6a6. The deep-cloning part of the note is still relevant.